test/mlir/krnl/krnl_getref_lowering.mlir

```diff
@@ -49,15 +47,14 @@ func @test_getref_lowering_dynamic(%arg0: memref<2x2xf32>) -> memref<2x?xf32> {
   // CHECK: %[[C13_I64:.+]] = llvm.mlir.constant(13 : i64) : !llvm.i64
   // CHECK: %[[C5_INDEX:.+]] = llvm.mlir.constant(5 : index) : !llvm.i64
   // CHECK: %[[C10_INDEX:.+]] = llvm.mlir.constant(10 : index) : !llvm.i64
-  // CHECK: [[MUL1:%.+]] = llvm.mul %[[C10_INDEX]], %[[C5_INDEX]] : !llvm.i64
-  // CHECK: llvm.mlir.null
-  // CHECK: llvm.mlir.constant(1 : index)
-  // CHECK: llvm.getelementptr
-  // CHECK: [[TYPE_SIZE_IN_BYTES:%.+]] = llvm.ptrtoint
+  // CHECK: %[[C1_INDEX:.+]] = llvm.mlir.constant(1 : index) : !llvm.i64
+  // CHECK: %[[MUL1:.+]] = llvm.mul %[[C5_INDEX]], %[[C10_INDEX]] : !llvm.i64
+  // CHECK: [[FLOAT_STAR:%.+]] = llvm.mlir.null : !llvm.ptr<float>
+  // CHECK: [[ELEM1:%.+]] = llvm.getelementptr [[FLOAT_STAR]][%[[MUL1]]] : (!llvm.ptr<float>, !llvm.i64) -> !llvm.ptr<float>
+  // CHECK: [[ELEM_SIZE:%.+]] = llvm.ptrtoint [[ELEM1]] : !llvm.ptr<float> to !llvm.i64
```

Comment:
Can you please explain how these changes happened?

Reply:
I believe these are due to a series of changes of changes for folding dynamic dimensions from late November. https://github.com/llvm/llvm-project/commits?author=sherhut

Reply:
Sounds good. This is fine with me.